packages/commonwealth/server/migrations/20240108174142-community-stake.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable(
+      'CommunityStakes',
+      {
+        community_id: {
```

Comment:
I am making an assumption here that each community can have multiple stakes by different (community_id, stake_id). But if this assumption is incorrect let me know.

Context:
https://commonxyz.slack.com/archives/C05CD20JCTU/p1704749134488389?thread_ts=1704748332.667029&cid=C05CD20JCTU

Reply:
yes all good here!